.github/workflows/main-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: main  
+run-name: Main Workflow
```

Comment:
What's the reason for using `run-name` here? I thought having different runs of this action have different names would be useful for debugging?

Reply:
ahh, I see yeah thats a good point.  I didn't realize that it changed all the actions in the actions tab to have the same name. I'll get rid of it